stdlib/public/core/InputStream.swift

```diff
@@ -59,9 +59,11 @@ public func readLine(strippingNewline: Bool = true) -> String? {
       }
     }
   }
+  let lineBytes = UnsafeMutableRawPointer(linePtr).bindMemory(
+    to: UTF8.CodeUnit.self, capacity: readBytes)
```

Comment:
You can also change `swift_stdlib_readLine_stdin` to return `unsigned char *`, seems like that will work better.